service/consul/environment.go

```diff
@@ -100,17 +100,19 @@ func newInstancers(l log.Logger, c gokitconsul.Client, co Options) (i service.In
 func newRegistrars(l log.Logger, registrationScheme string, c gokitconsul.Client, u ttlUpdater, co Options) (r service.Registrars, closer func() error, err error) {
 	var consulRegistrar sd.Registrar
 	for _, registration := range co.registrations() {
-		instance := service.FormatInstance(registrationScheme, registration.Address, registration.Port)
```

Comment:
Idiomatic go in this case is to do `registration := registration`

Reply:
I did not know that, thank you.

Fixed.